### ISSUE TITLE
Any plans for future support ? 

### ISSUE THREAD

**Issue by SushilShrestha** [satcar77/nepalicalender#3]
Are you planning to maintain the data or add new data for years to come ?

**Comment by satcar77** [comment]
Yes, I'll surely update the data if I get reminded like this. I have now added 2075 BS.

**Comment by SushilShrestha** [comment]
Hi Satcar, 
Are you like manually entering the data by hand or scrapping the data ? Just wanted to know about the source of data. I always wanted to work on the public apis for the events so just curious about how you get these contents.

**Comment by satcar77** [comment]
I scrape the data using Beautiful Soup
